js/index.js

```diff
@@ -1 +1,86 @@
 // Your code goes here
+const myInput1 = 
+document.querySelector('p');
+myInput1.addEventListener('click', (event) => {
+  myInput1.style.fontFamily = "Snell Roundhand, cursive";
+});
+
+const myInput2 = 
+document.querySelector('h1.logo-heading');
+myInput2.addEventListener('dblclick', (event) => {
+  myInput2.style.backgroundColor = "purple";
+});
+
+const myInput3 = 
+document.querySelector('h2');
+window.addEventListener('resize', (event) => {
+  myInput3.style.fontSize = "12px";
+  console.log(myInput3);
+});
+
+
+const myInput4 =
+document.querySelector("img");
+window.addEventListener('scroll', (event) => {
+  myInput4.style.border = "20px dotted pink";
+  console.log(myInput4);
+});
+
+const myInput5 =
+document.querySelector(".text-content");
+console.log(myInput5);
+window.addEventListener('mousemove', (event) => {
+  myInput5.style.color = "green";
+});
+
+const myInput6 =
+document.querySelector(".text-content");
+console.log(myInput6);
+window.addEventListener('mouseout', (event) => {
+  myInput6.style.color = "red";
+});
+
+const myInput7 =
+document.querySelector(".btn");
+console.log(myInput7);
+window.addEventListener('dragstart', (event) => {
```

Comment:
if you do 'dragstart' on the window, then if you drag any element in the window, the button changes color.